test/functionalTest/cases/1697_more_than_one_BadInput/more_than_one_BadInput.test

```diff
@@ -0,0 +1,135 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
more_than_one_BadInput.test probably is bad from a coding style rules. Maybe more_than_one_bad_input.test is better.